src/test/groovy/stix/datamarkings/TLPmarkingsSpec.groovy

```diff
@@ -0,0 +1,96 @@
+/**
+ * 
+ */
+package stix.datamarkings
+
+import org.skyscreamer.jsonassert.JSONAssert
+import org.skyscreamer.jsonassert.JSONCompareMode
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+
+import io.digitalstate.stix.common.StixInstant
+import io.digitalstate.stix.datamarkings.MarkingDefinition
+import io.digitalstate.stix.datamarkings.objects.Tlps
+import io.digitalstate.stix.json.StixParsers
+import io.digitalstate.stix.sdo.objects.Indicator
+import spock.lang.Shared
+import spock.lang.Specification
+
+/**
```

Comment:
- [ ] review usage of author javadoc tag as its starting to be deprecated in tooling like apache and gradle